rust/private/clippy.bzl

```diff
@@ -116,9 +116,14 @@ def _clippy_aspect_impl(target, ctx):
     if crate_info.is_test:
         args.add("--test")
 
+    valid_config_file_names = [".clippy.toml", "clippy.toml"]
```

Comment:
Does clippy care about the file name? Why can't this be arbitrary?

Reply:
Yes, clippy cares about the filename. https://github.com/rust-lang/rust-clippy/blob/cadb93b6de1b5ba5aca53c5c4c7987f3c09962a8/clippy_lints/src/utils/conf.rs#L220-L222

Reply:
```suggestion
    # Clippy itself looks for specifically named files so the same
    # enforcement is applied here.
    valid_config_file_names = [".clippy.toml", "clippy.toml"]
```